### ISSUE TITLE
[MacApp]  Cant paste text in raw part of BODY( POST method ) via context menu or hot key

### ISSUE THREAD

**Issue by anton-arsentyev** [a85/POSTMan-Chrome-Extension#2545]
1. Postman Version:  4.9.1 
 2. App (Chrome app or Mac app): Mac App
 3. OS details:   MacOs Sierra . 10.12.1 (16B2555)
 4. Is the Interceptor on and enabled in the app: NO
 5. Did you encounter this recently, or has this bug always been there: after last update

Steps to reproduce the problem:

Try to paste text in RAW part
via context menu or hot key
non saved request

**Comment by sdnts** [comment]
@anton-arsentyev Thanks for letting us know.
From my testing, I could not indeed right click inside the raw body editor (we're working on fixing this), but I could paste in text using Cmd/Ctrl + v. Are you not able to use this shortcut either?

**Comment by anton-arsentyev** [comment]
@madebysid No this shortcut doesn't work for me.
Works in URL field for example

**Comment by jtrunick** [comment]
same here, postman now unusable... too much data to enter by hand.

**Comment by JamieDixon** [comment]
I have this same issue. Unable to paste using `ctrl+v` and paste is greyed out in the context menu.

I just discovered that paste works from the menu bar at the top of OSX. I get the paste option there under "edit" 👍

**Comment by dominiceden** [comment]
I too have the same issue as @JamieDixon - there is no shortcut for pasting using `Cmd + V` - see screenshot attached. Very strange. I cannot assign the Postman application any shortcuts either in OS X as it's not a traditional Mac application.

Funnily enough I had the use of a 2015 MacBook Pro 13" recently and installed Postman, and `Cmd + V` did work out of the box.

![screen shot 2017-04-08 at 11 45 15 am](https://cloud.githubusercontent.com/assets/6367692/24828246/f4b60662-1c50-11e7-86de-d9025e652781.png)

**Comment by numaanashraf** [comment]
Hi @dominiceden, looks like you are using 'Postman for Chrome' on OSX. Could you share your OSX and Chrome versions? Postman cannot control the `Paste` shortcut and menu behavior due to limitations from the Chrome apps platform. We'll dig around for any known issues on OSX or Chrome that might be causing this for you. In the meantime, we recommend giving the more native 'Postman for Mac' a shot (https://www.getpostman.com/apps) as Chrome apps are soon to be deprecated by Google.

**Comment by dominiceden** [comment]
@numaanashraf Sure:

* Chrome Version 58.0.3029.110 (64-bit)
* Mac OS Sierra 10.12.5

Thanks for the headsup on the native app, I'll download that - wasn't aware.

**Comment by sleyzerzon** [comment]
I too just experienced this issue, would be nice to have it working.  In Windows it used to work fine.  Thanks.

I'm on MacBook Pro macOS Sierra v. 10.12.6
Postman details:

 Postman for Chrome
Version 5.1.2
OS X / x86-64
Chrome 59.0.3071.115

**Comment by rogerprz** [comment]
unable to use the text portion. need to paste raw :(

**Comment by prashantagarwal** [comment]
@rogercodes1  Can you update to the latest Postman app and check if the issue still persists.

**Comment by prashantagarwal** [comment]
Tracking in #4759

**Comment by charlesr1971** [comment]
This doesn’t work on iOS 16.6, Mobile Safari? And the problem is that on the web UI, there is no EDIT menu. So there really isn’t any way to `copy/cut/paste` anything inside the Body textarea etc
Also the keyboard doesn’t appear when you tap inside the textarea. 
Please can you clean up the experience for mobile Safari users. Currently this part of Postman is unusable. 🤔

**Comment by vtcajones** [comment]
Yes, editing or adding JSON to a request is practically impossible on mobile safari. Otherwise love the app!

**Comment by shamasis** [comment]
For a moment I freaked out seeing v4.9 in the main post! Then I realised its appended to a 8y old post. 😂😂

@vtcajones Are you on iPad? (If you're on iPhone then that's some deep love for Postman and I salute you for that.)

@codenirvana it would be cool to see if this a low hanging thing or not. Usually context menu trigger fixes are simple - but this is Monaco editor so, anything can get complex. 😂
